.gitignore

```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
+
+.prettierrc.json
+package-lock.json
```

Comment:
We want to upload `yarn.lock`. The `package-lock.json` should never exist in the directory(even locally!)